sw/device/silicon_creator/manuf/lib/sram_program.ld

```diff
@@ -29,14 +29,22 @@ _dv_log_offset = 0x10000;
 ENTRY(sram_start);
 
 SECTIONS {
-  .text ORIGIN(ram_main) + _static_critical_size : ALIGN(4) {
+  /* The ELF SRAM program loader will automatically read back the data of the section
+   * containing the entry point to verify that it is correct. The expectation is that
+   * this section is very small and only contains the code that computes the CRC of
+   * everything else. For this reason, we separate the sram start+crt code from the
+   * rest. */
+  .sram_start ORIGIN(ram_main) + _static_critical_size : ALIGN(4) {
```

Comment:
Isn't the entire CRC calculation code included in `.sram_start`? Why is `.crt` also included in this section?

Reply:
Yes you are right, not sure why I included `.crt`, thanks